spec/services/document_unpublishing_service_spec.rb

```diff
@@ -0,0 +1,72 @@
+# frozen_string_literal: true
+
+require "spec_helper"
+
+RSpec.describe DocumentUnpublishingService do
+  describe "#retire" do
+    let(:document) { create(:document) }
+
+    before do
+      @explanatory_note = "The document is out of date"
```

Comment:
Could this be a let?

Reply:
That's interesting - you mean like `let(:explanatory_note) { "blah" }`?

Reply:
Exactly like that.

Reply:
Just tried it again, and it's working now. I must have had a syntax error earlier.

Reply:
Fixed up.

Reply:
It works for me (removing the @ everywhere).

Reply:
Just missed your comment :-).